ground/uavobjgenerator/uavobjectparser.cpp

```diff
@@ -117,6 +118,26 @@ bool fieldTypeLessThan(const FieldInfo* f1, const FieldInfo* f2)
 }
 
 /**
+ * Returns a human-meaningful error message
+ * @param fileName The filename we found an error in
+ * @param errMsg A human readable description of what's wrong
+ * @param errorLine The line number where this problem occurred
+ * @param errorCol The column number
+ * @returns A formatted error string.
+ */
+QString UAVObjectParser::genErrorMsg(QString& fileName,

```

Comment:
I agree-- I like it that way.

Unfortunately, that whole file, and all of the QT code we have puts the referencing operator next to the type name.